components/home-page/bal-action/action-bbe/working-with-databases.md

```diff
@@ -24,14 +24,14 @@ service / on new http:Listener(8080) {
         self.db = check new (host, user, password, database, port);
     }
 
-    resource function get albums() returns Album[]|error? {
+    resource method get albums() returns Album[]|error? {
```

Comment:
Should not be changed in the code snippets

Reply:
Fixed in https://github.com/ballerina-platform/ballerina-dev-website/pull/6013.